app.py

```diff
@@ -0,0 +1,87 @@
+import base64
+from flask import Flask, request, render_template
+from flask_cors import cross_origin
+from image_utils import smile_detecting, face_as_net_input, extract_faces
+from keras.models import load_model
+import tensorflow as tf
+import argparse
+import numpy as np
+import cv2
+
+# creates a Flask application, named app
+app = Flask(__name__)
+
+
+cascpath = "haarcascade_frontalface_default.xml"
+modelpath = "model.hdf5"
```

Comment:
Why initialize here if you pass the path from cli?

Reply:
done